x/distribution/alias.go

```diff
@@ -59,6 +59,14 @@ var (
 	NewQueryDelegatorParams                   = keeper.NewQueryDelegatorParams
 	NewQueryDelegatorWithdrawAddrParams       = keeper.NewQueryDelegatorWithdrawAddrParams
 	DefaultParamspace                         = keeper.DefaultParamspace
+	RegisterInvariants                        = keeper.RegisterInvariants
+	AllInvariants                             = keeper.AllInvariants
+	NonNegativeOutstandingInvariant           = keeper.NonNegativeOutstandingInvariant
+	CanWithdrawInvariant                      = keeper.CanWithdrawInvariant
+	ReferenceCountInvariant                   = keeper.ReferenceCountInvariant
+	CreateTestInputDefault                    = keeper.CreateTestInputDefault
+	CreateTestInputAdvanced                   = keeper.CreateTestInputAdvanced
+	TestAddrs                                 = keeper.TestAddrs
```

Comment:
What is `TestAddrs`? Anything pertaining to tests?

Reply:
yes, they are predefined addresses to test with.

Reply:
In that case I don't think they should be defined or set here.

Reply:
Could you be more specific? How better should this be organized?

Reply:
I just don't think they should be set or defined in "core" code. i.e. keep them in tests files or test utilities.

Reply:
so this is defined in a "test" file (aka `keeper/test_common.go`) I'm kinda peeved at all the code dup between the this test_common.go file across all the modules... but maybe it's worth refactoring into a new testing utilities package like `x/testing_utilities` - what you think of that?